### DISCUSSION TITLE
How to remove cars default location (Tesla Fremont Factory) after factory reset?

### DISCUSSION THREAD

**Discussion by techsurrusr** [teslamate-org/teslamate#2648] [Q&A] [answered]
Recently had to perform a factory reset of my car and noticed then that the car logged a quick jump to the default location when the car has been factory reset which is Tesla Fremont Factory. When quickly looking for entry points in Grafana for Drivers, Trip, Locations, there aren't any entry points. Since I want to delete this entry, which database table would a be looking at to find the entry to be deleted?

Based on the above little issue, are there any plans to include a small maintenance section in TeslaMate to make such deletion easier (the above issue I hardly can be the first to have and which could be an example of a ready-made query for quick deletion from database)?

**Comment by cwanja** [comment]
- Drives are from `drives`
- Locations are from `addresses`
- Trip is a collection of other dashboards

I feel like it is a tough stretch to get this introduced into the front end of TeslaMate given that some have it public facing and potentially unsecured. No one's fault, but that owner. But TeslaMate got caught up in a separate issue earlier this year.

**Comment by DrMichael** [comment, marked as answer]
Have a look at `positions` table...

**Reply by techsurrusr** [reply]
@DrMichael Thanks I'll going to see what I find in the `positions` table as neither `driver` or `addresses` include anything related to the freemont address or as you pointed out that it most likely is related to the default position.

**Reply by techsurrusr** [reply]
@DrMichael Brief feedback on my small issue. To be able to delete the Palo Alto reference using the filter query "longitude = '-122.150604'" initially gave me an error that deleting on table "positions" violates foreign key constraint in "charging_processes_position_id_fkey". By first deleting the referenced "charging_processes" key id first, I could then proceed with the deleting the position without getting any errors.

Thanks for the help pointing me in the right direction what to look for.

**Reply by tomS3210** [reply]
hello @techsurrusr , i have the same issue (positions" violates foreign key constraint) when i want to delete the Position with Palo Alto. How di you "delete the referenced "charging_processes" key id first" ?

Thanks

**Reply by tomS3210** [reply]
Finally I looked in a backup of the database with a text editor what was the ID of the line to delete in charging_processes then I deleted it with the command:

`delete from charging_processes where id = 707;`

I was then able to delete the last position:

`delete from positions where id = 9001000;`
